addons/jam/CfgWeapons.hpp

```diff
@@ -6,76 +6,73 @@ class CfgWeapons {
     class Rifle_Short_Base_F;
 
     class mk20_base_F: Rifle_Base_F {
-        magazineWell[] = {"CBA_556x45_STANAG", "CBA_556x45_STANAG_L", "CBA_556x45_STANAG_XL", "CBA_556x45_STANAG_2D"};
+        magazineWell[] = {"STANAG_556x45", "CBA_556x45_STANAG", "CBA_556x45_STANAG_L", "CBA_556x45_STANAG_XL", "CBA_556x45_STANAG_2D"};
```

Comment:
Maybe we should just use += ? 
Currently we also break any mod that gets loaded before CBA and tries to add magwells.

Reply:
Can you even += with all the inheritance going on?

Reply:
Sure. If a subclass just uses `=` that will not be of any use. But that would also break our `=` in the base class so that doesn't matter.
Just need to make sure that requiredAddons is correct, which is already correct.

Reply:
+= also fails horribly if the magazineWell token is inherited. I guess I will have to check all classes, but thankfully I have the config dumps.

Reply:
wasn't += built to work with inheritance? so that you can just add elements instead of overwriting..

Reply:
No. It breaks for inherited tokens, but it works for different config patches adding to the same class.

Reply:
It even breaks in funny ways like adding the same array element twice overwriting the old ones. It goes completely bonkers. It is a terrible idea to rely on it for this stuff, but whatever rocks there boat. I will quickly eat something and then do what we need to in CBA and ACE PRs. Gimme a minute.